botocore/docs/bcdoc/style.py

```diff
@@ -227,7 +227,7 @@ def sphinx_reference_label(self, label, text=None):
         else:
             self.doc.write(text)
 
-    def end_a(self):
+    def end_a(self, next):
```

Comment:
Let's make next optional here:
```suggestion
    def end_a(self, next=None):
```